packages/lsp/package.json

```
@@ -29,7 +29,7 @@
     "style/index.js"
   ],
   "scripts": {
-    "build": "jlpm build:schema && tsc -b",
+    "build": "npm run build:schema && tsc -b",
```

Removing the build schema step in `build` would help. Since we already have a test for obsolete schemes on CI, I think we don't need to re-generate these files on every build.
```suggestion
    "build": "tsc -b",
```